src/ForeignController.sol

```diff
@@ -16,11 +19,14 @@ interface ICCTPLike {
         bytes32 mintRecipient,
         address burnToken
     ) external returns (uint64 nonce);
+    function localMinter() external view returns (ICCTPTokenMinterLike);
 }
 
-contract ForeignController is AccessControl {
+interface ICCTPTokenMinterLike {
+    function burnLimitsPerMessage(address) external view returns (uint256);
+}
```

Comment:
I'd move `ICCTPTokenMinterLike` and `ICCTPLike` to separate files

Reply:
I did this, but I made another task to just use the project interfaces: https://github.com/circlefin/evm-cctp-contracts/tree/master/src/interfaces